cdc/sink/codec/config_test.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package codec
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/pingcap/tiflow/pkg/config"
+	cerror "github.com/pingcap/tiflow/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewConfig(t *testing.T) {
+	c := NewConfig("fake-protocol")
+	require.Equal(t, c.protocol, "fake-protocol")
+	require.Equal(t, c.maxMessageBytes, config.DefaultMaxMessageBytes)
+	require.Equal(t, c.maxBatchSize, defaultMaxBatchSize)
+	require.Equal(t, c.enableTiDBExtension, false)
+	require.Equal(t, c.avroRegistry, "")
+}
+
+func TestConfigApplyValidate(t *testing.T) {
```

Comment:
Could you add a multi-endpoint tests?

`kafka://127.0.0.1:9092,127.0.0.1:9091,127.0.0.1:9090/...`